### PR TITLE
 Added details about decay heat function units

### PR DESCRIPTION
**Pull request by gwenchee**
The decay heat function units is in MW/material object total mass. These lines were added for the sake of clarity for future users.

### CODE REVIEW

File: pyne/material.pyx [hunk 1]
```diff
@@ -608,7 +608,9 @@ cdef class _Material:
 
 
     def decay_heat(self):
-        """This provides the decay heat using the comp of the the Material.
+        """This provides the decay heat using the comp of the the Material. It
+        assumes that the composition of material is given in units of [grams]
+        and returns decay heat in units of [MW/material object total mass].
```
Comment (scopatz) [inline pyne/material.pyx]: I think the decay heat is returned in  units of MW: https://github.com/pyne/pyne/blob/develop/src/material.h#L237

**Comment by scopatz** [comment]
Thanks for putting this in @gwenchee!

File: pyne/material.pyx (hunk 1 above)
Comment (gwenchee) [reply pyne/material.pyx]: I think line 237 is referring to the activity function, https://github.com/pyne/pyne/blob/develop/src/material.h#L239 says that the decay heat function returns dose per gram, which is synonymous with [MW/material object total mass in grams]. I will correct my PR to include grams.

Comment (scopatz) [reply pyne/material.pyx]: Hi @gwenchee - doxygen/javadoc style docstrings are placed above the function they are documenting, not below.  So 237 is the correct line.  Also, if you see 239 has units of mrem/h, which is more appropriate for dose, than decay heat.

**Comment by katyhuff** [comment]
@gwenchee this looks stalled... I think a pr update was among next steps?

File: pyne/material.pyx (hunk 1 above)
Comment (gwenchee) [reply pyne/material.pyx]: Yup that makes sense. I changed it to reflect that the unit is [MW]

**Comment by gwenchee** [comment]
I updated my commit to reflect Anthony's comments. It should be ready for merge now.

**Comment by scopatz** [comment]
Hi @gwenchee can your merge from develop please to get the latest commits so that CI will pass

**Comment by gwenchee** [comment]
done! @scopatz

**Comment by scopatz** [comment]
Thanks @gwenchee!

**Pull request merged**